docs/app/partials/home.tmpl.html

```diff
@@ -50,43 +50,24 @@ <h2 class="md-headline">The Latest Material Design</h2>
     <p>
       The latest update to Material Design
       (<a href="https://www.youtube.com/playlist?list=PLJ21zHI2TNh-rX-Xr_xi9KIEcbdee_1Ah" target="_blank" rel="noopener">video playlist</a>)
-      was announced at Google I/O in May, 2018
+      was announced at Google I/O in May 2018
       (<a href="https://design.google/library/io-2018-our-definitive-guide-design/" target="_blank" rel="noopener">recap blog post</a>).
       For an implementation of this new
       <a href="https://material.io/design" target="_blank" rel="noopener">Material Design Specification</a>,
       please see the <a href="https://github.com/angular/components" target="_blank" rel="noopener">Angular Material</a>
       project which is built for <a href="https://angular.io" target="_blank" rel="noopener">Angular</a>
       developers.
     </p>
-    <h2 id="long-term-support" class="md-headline">Long Term Support</h2>
+    <h2 id="long-term-support" class="md-headline">End-Of-Life</h2>
     <p>
-      AngularJS Material has entered a Long Term Support (LTS) period which ends on December 31st,
-      2021. This date aligns with the
-      <a href="https://docs.angularjs.org/misc/version-support-status" target="_blank" rel="noopener">
-      AngularJS LTS</a> period. Explore End Of Life (EOL) options
-      <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429" target="_blank" rel="noopener">here</a>.
-    </p>
-    <p>
-      We now focus exclusively on providing fixes to bugs, in the 1.2.x branch of the library, that
-      satisfy at least one of the following criteria:
-    </p>
-    <ul>
-      <li>We receive a report of, or detect, a security weakness which is then verified</li>
-      <li>
-        A major browser releases a version that will cause current production applications
-        to stop working
-      </li>
-      <li>
-        The <a href="https://jquery.com/">jQuery library</a> releases a version that will cause
-        current production applications to stop working
-      </li>
-    </ul>
-    <p>
-      AngularJS Material 1.1.x will get a new version, if and only if, we verify a new severe
-      security weakness.
+      AngularJS Material support has officially ended as of January 2022. This date aligns with the end of
```

Comment:
Could we make this content match the text in security.md (which matches the main AngularJS text)? In particular, I think it's important to point to the actively supported thing.

Reply:
Sure, I can do that. The only reason that I didn't, was that it seemed repetitive since the previous 2 paragraphs both discuss and link to Angular Material.